edx_sga/sga.py

```diff
@@ -747,9 +748,17 @@ def student_state(self):
         Returns a JSON serializable representation of student's state for
         rendering in client view.
         """
+        graded = None
         submission = self.get_submission()
+
         if submission:
             uploaded = {"filename": submission["answer"]["filename"]}
+            student_item = SubmissionsStudent.objects.get(id = submission.get("student_item"))
+            score = submissions_api.get_score(student_item.student_item_dict)
+            if score:
+                score = score.get("points_earned")
+                graded = {"score": score, "comment": force_str(self.comment)}
+
```

Comment:
We are doing a redundant thing here. e.g. We are trying to get the score with `score = submissions_api.get_score(student_item.student_item_dict)`. However, In the code that you removed we were calling [`self.score`](https://github.com/mitodl/edx-sga/blob/master/edx_sga/sga.py#L688-L693) which is already a property of this class that uses `@reify` decorator.